client/README.md

```diff
@@ -8,6 +8,10 @@
 - From this directory, `npm run setup` to install front end dependencies and create an `.env` file
 - get a [Mapbox](https://account.mapbox.com/auth/signin/) API token and add that to your `.env` file as the `MAPBOX_TOKEN`
 - get the `API_URL` from a team member and add that to your `.env` file. Alternatively, you can [bring up](https://github.com/hackforla/311-data/blob/dev/docs/server_setup.md) your own local server and use that.
+- please replace `"postinstall": "sh -c \"are-you-es5 check -r . | tail -n 2 | head -n 1 > ./node_modules_non_es5 \""` from line 67 
```

Comment:
Did you see my findings in https://github.com/hackforla/311-data/pull/1207#issuecomment-1137923083? The postinstall check is not needed anymore, so I prefer to get rid of it instead of making all our contributors manually remove it every time.